Cargo.toml

```diff
@@ -29,6 +33,8 @@ aws-lc-rs = { version = "1", optional = true }
 anyhow = "1"
 clap = { version = "4", features = ["derive"] }
 rustls-pki-types = "1"
+hex = "0.4.3"
```

Comment:
what is it for?

Reply:
removed. probably in the previous try-n-error state, I added those.